node_modules/col-assets/lib/api.js

```diff
@@ -208,8 +209,14 @@ var incrementViewsIfRequired = function(asset, incrementViews, callback) {
       log.error({'err': err, 'id': id}, 'Failed to increase the views on an asset');
       return callback({'code': 500, 'msg': err.message});
     }
+    ActivitiesAPI.createActivity(ctx.course, ctx.user, 'view_asset', id, CollabosphereConstants.ACTIVITY.OBJECT_TYPES.ASSET, null, null, function(err) {
+      if (err) {
+        log.error({'user': ctx.user, 'err': err}, 'Failed to create activity \'view_asset\' for user');
+        return errorCallback(err);
```

Comment:
`errorCallback` doesn't appear to be defined anywhere. Should just be `return callback(err);`.